index.js

```diff
@@ -1,19 +1,19 @@
 'use strict'
 
 const NODEJS_VERSION = parseInt(process.version.slice(1).split('.')[0], 10)
-const STACKTRACE_OFFSET = NODEJS_VERSION && NODEJS_VERSION > 6 ? 4 : 5
-const LINE_OFFSET = 7
+const DEFAULT_STACKTRACE_OFFSET = NODEJS_VERSION && NODEJS_VERSION > 6 ? 4 : 5
+const DEFAULT_LINE_OFFSET = 7
 const { symbols } = require('pino')
 const { asJsonSym } = symbols
 
-function traceCaller (pinoInstance) {
+function traceCaller (pinoInstance, {stackTraceOffset = 0, lineOffset = 0} = {}) {
```

Comment:
imho no need to change the semantic of the function, see #18 for a more generic approach.

Reply:
This Pr allows the user to dynamically tweak based on the plugin or plugins they are going to use. 
#18 implies the bug is only happening because of `node_modules/pino`